common/common-api/src/main/java/org/ow2/proactive/core/properties/PASharedProperties.java

```diff
@@ -54,7 +54,26 @@
     FAILED_LOGIN_RENEW_MINUTES("pa.shared.failed.renew.minutes", PropertyType.INTEGER, "10"),
 
     /** Key used when decrypting properties */
-    PROPERTIES_CRYPT_KEY("pa.shared.properties.crypt.key", PropertyType.STRING, "activeeon");
+    PROPERTIES_CRYPT_KEY("pa.shared.properties.crypt.key", PropertyType.STRING, "activeeon"),
+
+    /* ***************************************************************** */
+    /* ******************* SCHEDULER BACKUP PROPERTIES ******************* */
+    /* ***************************************************************** */
+
+    /** if backup is enabled */
+    SERVER_BACKUP("pa.server.backup", PropertyType.BOOLEAN, "false"),
+
+    /** cron expression which defines when backuop should be performed */
+    SERVER_BACKUP_PERIOD("pa.server.backup.period", PropertyType.STRING, "0 * * * *"),
```

Comment:
you could change the default value here also

Reply:
ok